### DISCUSSION TITLE
Send lora radio to sleep on sensor node with deep sleep enabled?

### DISCUSSION THREAD

**Discussion by ThePradox** [timmbogner/Farm-Data-Relay-System#241] [Ideas]
Hi and thanks for the great project.

I may be missing something but is there a reason for not calling ```radio.sleep()``` in [fdrs_node.h](https://github.com/timmbogner/Farm-Data-Relay-System/blob/d6e63eb50bb96874b7b270cf9e45f29ff67fb22b/src/fdrs_node.h#L278)?

I tested it on my simple sensor node with an sx1262 and found no problems so far. I lack the equipment to measure if it makes a huge difference in power consumption, but most radio vendors advertise very low sleep currents.

**Comment by timmbogner** [comment]
Sorry, a bit of a late response.
I've been away from things and can't recall definitively, being honest. My recollection is that, as you guessed, it doesn't make a big difference.
